apps/system/test/unit/app_transition_controller_test.js

```diff
@@ -211,15 +211,6 @@ suite('system/AppTransitionController', function() {
     assert.isTrue(stubReviveBrowser.called);
   });
 
-  test('Handle opening of callscreen window', function() {
-    var app1 = new MockAppWindow(fakeAppConfig1);
-    var acn1 = new AppTransitionController(app1);
-    var stubSetOrientation = this.sinon.stub(app1, 'setOrientation');
-    app1.isCallscreenWindow = true;
-    acn1.handle_opening();
-    assert.isTrue(stubSetOrientation.calledOnce);
-  });
```

Comment:
I think we actually want this.

Currently while on a browser tab, in landscape, if you receive a call it'll open the callscreen in landscape first, then rotate to portrait.
We want to rotate to portrait first, then show the callscreen.